packages/ts-docs-gen/src/file-manager.ts

```diff
@@ -1,81 +1,81 @@
-import { Contracts } from "ts-extractor";
 import { MarkdownGenerator } from "@simplrjs/markdown";
 import * as path from "path";
 
-import { FileManagerBaseBase } from "./abstractions/file-manager-base";
-import { RenderItemOutputDto } from "./contracts/render-item-output-dto";
-import { FileOutputDto } from "./contracts/file-output-dto";
+import { FileManager as FileManagerInterface } from "./contracts/file-manager";
 import { Helpers } from "./utils/helpers";
+import { PluginResult } from "./contracts/plugin";
+import { FileResult } from "./contracts/file-result";
+import { GeneratorHelpers } from "./generator-helpers";
 
 interface OutputData {

```

Comment:
Is this interface used?

Reply:
It was used before. For now leave it. I think after `ts-extractor` update it will be removed.